core/src/test/kotlin/org/evomaster/core/output/naming/RestActionNamingStrategyTest.kt

```diff
@@ -92,34 +211,166 @@ class RestActionNamingStrategyTest {
     @Test
     fun testResponseNamedWithMultipleFaults() {
         val faults = listOf(DetectedFault(FaultCategory.GQL_ERROR_FIELD, "items"), DetectedFault(FaultCategory.HTTP_INVALID_LOCATION, "items"), DetectedFault(FaultCategory.HTTP_STATUS_500, "items"))
-        val eIndividual = getEvaluatedIndividualWithFaults("/items", 500, faults)
-        val languageConventionFormatter = LanguageConventionFormatter(OutputFormat.PYTHON_UNITTEST)
-
+        val restAction = getRestCallAction()
+        val eIndividual = getEvaluatedIndividualWithFaults(restAction, faults, 500)
         val solution = Solution(singletonList(eIndividual), "suitePrefix", "suiteSuffix", Termination.NONE, emptyList(), emptyList())
 
-        val namingStrategy = RestActionTestCaseNamingStrategy(solution, languageConventionFormatter)
+        val namingStrategy = RestActionTestCaseNamingStrategy(solution, pythonFormatter, EMConfig())
 
         val testCases = namingStrategy.getTestCases()
         assertEquals(1, testCases.size)
         assertEquals("test_0_GET_on_items_showsFaults_100_102_301", testCases[0].name)
     }
 
-    private fun getEvaluatedIndividualWith(path: String, statusCode: Int = 200): EvaluatedIndividual<RestIndividual> {
-        return getEvaluatedIndividualWithFaults(path, statusCode, emptyList())
+    @Test
+    fun testIndividualUsingSql() {
+        val restAction = getRestCallAction()
+        val eIndividual = getEvaluatedIndividualWith(restAction, true)
+        val solution = Solution(singletonList(eIndividual), "suitePrefix", "suiteSuffix", Termination.NONE, emptyList(), emptyList())
+
+        val namingStrategy = RestActionTestCaseNamingStrategy(solution, javaFormatter, EMConfig())
+
+        val testCases = namingStrategy.getTestCases()
+        assertEquals(1, testCases.size)
+        assertEquals("test_0_getOnItemsReturnsEmptyUsingSql", testCases[0].name)
+    }
+
+    //    @Test// TODO check PhG Gene#isLocallyValid with JP/AA
+    fun testIndividualUsingMongo() {
```

Comment:
@arcuri82 @jgaleotti 
I'm having trouble with this test case. When executing I get the following exception:
```
java.lang.AssertionError: Assertion failed

	at org.evomaster.core.Lazy.assert(Lazy.kt:16)
	at org.evomaster.core.search.gene.Gene.doInitialize(Gene.kt:176)
	at org.evomaster.core.search.action.Action.doInitialize(Action.kt:75)
	at org.evomaster.core.search.Individual.doInitialize(Individual.kt:198)
	at org.evomaster.core.TestUtils.doInitializeIndividualForTesting(TestUtils.kt:30)
	at org.evomaster.core.TestUtils.doInitializeIndividualForTesting$default(TestUtils.kt:28)
	at org.evomaster.core.output.naming.RestActionNamingStrategyTest.getEvaluatedIndividualWithFaults(RestActionNamingStrategyTest.kt:337)
	at org.evomaster.core.output.naming.RestActionNamingStrategyTest.getEvaluatedIndividualWith(RestActionNamingStrategyTest.kt:294)
	at org.evomaster.core.output.naming.RestActionNamingStrategyTest.getEvaluatedIndividualWith(RestActionNamingStrategyTest.kt:286)
	at org.evomaster.core.output.naming.RestActionNamingStrategyTest.getEvaluatedIndividualWith$default(RestActionNamingStrategyTest.kt:285)
	at org.evomaster.core.output.naming.RestActionNamingStrategyTest.testIndividualUsingMongo(RestActionNamingStrategyTest.kt:241)
```

Which apparently has something to do with how genes are initialized. The `isLocallyValid` call fails for some reason I can't understand.
Any idea what I could look into to fix it?